sympy/assumptions/ask.py

```diff
@@ -25,1124 +26,213 @@ class AssumptionKeys:
 
     @memoize_property
     def hermitian(self):
-        """
-        Hermitian predicate.
-
-        ``ask(Q.hermitian(x))`` is true iff ``x`` belongs to the set of
-        Hermitian operators.
-
-        References
-        ==========
-
-        .. [1] http://mathworld.wolfram.com/HermitianOperator.html
-
-        """
-        # TODO: Add examples
-        return Predicate('hermitian')
+        from .predicates.sets import HermitianPredicate
+        return HermitianPredicate('hermitian')
```

Comment:
Passing string names into these classes feels useless to me - would anyone ever use `HermitianPredicate('not hermitian')`?